charts/zeebe-benchmark/templates/leader-balancing-cron.yaml

```
@@ -0,0 +1,23 @@
+{{- if .Values.leaderBalancing.enabled -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: leader-balancer
+  labels:
+    {{- include "zeebe-benchmark.labels" . | nindent 4 }}
+spec:
+  schedule: {{ .Values.leaderBalancing.schedule | quote }}
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  startingDeadlineSeconds: 120
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - image: "curlimages/curl:7.87.0"
+              name: curator
```

:wrench: should be `curl` or `rebalancer` or something
```suggestion
              name: curl
```